### ISSUE TITLE
Group (sponsored) membership

### ISSUE THREAD

**Issue by benjie** [somakeit/members-area#14]
Allow groups of members where one payment (e.g. with identifier: G000001) can cover multiple members.

Allow groups to have one (or more, in future) administrator who can choose which members are funded.

Group members still sign up as individual So Make It members (with voting rights) but the group pays their subscription.

**Comment by antonpiatek** [comment]
I thought corporate members would have limited voting rights to prevent a company from exerting control over the group.

**Comment by benjie** [comment]
Correct, in fact the company has no voting rights at all under this scheme, they simply sponsor members (like a grant). Sure their employees can vote as So Make It members but it wouldn't be ethical to require employees to vote a particular way.

**Comment by jeremybennett** [comment]
I suggested it to Benje to simplify things for companies who want to pay for their staff to be members. I agree absolutely that companies should have no rights - it should just be an accounting convenience. Really just a way to allow someone to pay for multiple members. Indeed it would work just as well for family membership.
